app/views/layouts/document_upload.html.erb

```diff
@@ -61,6 +61,15 @@
                 <% end %>
               </div>
             </div>
+
+            <% if @form.errors[:document].any? %>
+              <div class="form-group form-group--error">
+                <p class="text--error"><i class="icon-warning"></i>
+                  <%= @form.errors.full_messages.first %>
+                </p>
+              </div>
+            <% end %>
```

Comment:
`file_field` doesn't seem to include errors in the way we do on our other form fields (see `VitaMinFormBuilder`) so i had to add them in

Reply:
Would be good to add screenshot